src/Compilers/CSharp/Portable/BoundTree/Expression.cs

```diff
@@ -119,7 +120,7 @@ internal static IArgument CreateArgumentOperation(ArgumentKind kind, IParameterS
                     //TODO: https://github.com/dotnet/roslyn/issues/18722
```

Comment:
The new `IInvalidInvocationExpression` will only contains a list of argument values provided to the call based on the design, so we will likely fix it in a similar fashion as how we handle it in this PR: decide if there's any error early, diving into the business of deriving argument values only necessary (only for non-`IInvalidInvocationExpression` ).